ClimbingGearBackend/Controllers/UserGearController.cs

```diff
@@ -38,11 +37,11 @@ public async Task<ActionResult<IEnumerable<UserGearDTO>>> GetUserGear(long userI
 
     // GET: api/UserGear/5
     [HttpGet("{id}")]
-    public async Task<ActionResult<UserGearDTO>> GetSingleUserGear(long userGearId)
+    public async Task<ActionResult<UserGearDTO>> GetUserGear(long id)
```

Comment:
Should there be some validation that the authenticated user has permission to view this particular item? Similarly for the mutation methods below.

Reply:
Yes, I am going to add that. The way that piece of code works is that the id is grabbed from the route so `api/usergear/5` would pass `5` as a parameter to the method. If the parameter names for don't match it won't get passed to the method.

Reply:
Often, things you intend to add in the future which are necessary for completeness like this should be noted with a `TODO` comment. This way reviewers know you have thought about it and intend to return to it.